cmd/argocd/commands/users.go

```diff
@@ -0,0 +1,71 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/argoproj/argo-cd/errors"
+	argocdclient "github.com/argoproj/argo-cd/pkg/apiclient"
+	"github.com/argoproj/argo-cd/server/users"
+	"github.com/argoproj/argo-cd/util"
+	"github.com/argoproj/argo-cd/util/settings"
+	"github.com/spf13/cobra"
+)
+
+func NewUsersCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
+	var command = &cobra.Command{
+		Use:   "users",
+		Short: "Manage users",
+		Run: func(c *cobra.Command, args []string) {
+			c.HelpFunc()(c, args)
+			os.Exit(1)
+		},
+	}
+	command.AddCommand(NewUsersChangePasswordCommand(clientOpts))
+	return command
+}
+
+func NewUsersChangePasswordCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
+	var (
+		CurrentPassword string
+		NewPassword     string
+	)
+	var command = &cobra.Command{
+		Use:   "change-password USERNAME",
+		Short: "Change User Password",
+		Run: func(c *cobra.Command, args []string) {
+			if len(args) == 0 {
+				c.HelpFunc()(c, args)
+				os.Exit(1)
+			}
+
+			if CurrentPassword == "" {
+				CurrentPassword = settings.ReadAndConfirmPassword("Current Password")
```

Comment:
We should not require that the user type in their current password twice. Password confirmation should be done only for the new one.